src/context.rs

```diff
@@ -100,9 +100,9 @@ mod private {
     use super::*;
     pub trait Sealed {}
 
-    impl<'buf> Sealed for AllPreallocated<'buf> {}
-    impl<'buf> Sealed for VerifyOnlyPreallocated<'buf> {}
-    impl<'buf> Sealed for SignOnlyPreallocated<'buf> {}
+    impl Sealed for AllPreallocated<'_> {}
+    impl Sealed for VerifyOnlyPreallocated<'_> {}
+    impl Sealed for SignOnlyPreallocated<'_> {}
```

Comment:
Honesty, these feel like useless churn but whatever, I don't care.

Reply:
Clippy requires we change them.

Reply:
I mean, I'd consider turning off that lint. Or maybe, if it was off by default, I wouldn't turn it on.